virtcontainers/sandbox.go

```diff
@@ -206,7 +205,7 @@ type Sandbox struct {
 	hypervisor hypervisor
 	agent      agent
 	storage    resourceStorage
-	network    network
+	network    Network
```

Comment:
@sameo why did this need to be made public?

Reply:
I'm preparing for moving the network code into its own package.